plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -297,6 +297,7 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
             message("codemodernizer.notification.info.transformation_stop.title"),
             message("codemodernizer.notification.info.transformation_stop.content"),
             project,
+            listOf(displayFeedbackNotificationAction())
```

Comment:
This is an unrelated change to the VFS refresh. Probably this should have been added in a separate PR.